### ISSUE TITLE
intellisense settings is innacurate

### ISSUE THREAD

**Issue by badsector998** [badsector998/raspi-pico-template#1]
example from `${PICO_SDK_PATH}/examples/blink/`

<details><summary>CPP Config .vscode</summary>
<p>

```json
{
    "configurations": [
        {
            "name": "Win32",
            "includePath": [
                "${workspaceFolder}/**",
                "D:\\Tools\\vsarm-rpi-pico\\pico-sdk\\**"
            ],
            "defines": [
                "_DEBUG",
                "UNICODE",
                "_UNICODE"
            ],
            "windowsSdkVersion": "10.0.22000.0",
            "cStandard": "c17",
            "cppStandard": "c++17",
            "intelliSenseMode": "linux-gcc-x64",
            "compilerPath": "C:/mingw/MinGW/bin/gcc.exe",
            "configurationProvider": "ms-vscode.cmake-tools"
        },
        {
            "name": "ARM-Pico",
            "includePath": [
                "${workspaceFolder}/**"
            ],
            "defines": [
                "_DEBUG",
                "UNICODE",
                "_UNICODE"
            ],
            "windowsSdkVersion": "10.0.22000.0",
            "cStandard": "c17",
            "cppStandard": "c++17",
            "intelliSenseMode": "linux-gcc-x64",
            "compilerPath": "C:/mingw/MinGW/bin/gcc.exe"
        }
    ],
    "version": 4
}
```

</p>
</details> 

- Settings above works perfectly in vscode
- If used outside `$PICO_SDK_PATH` it's not working

Issues might related to:
- Compiler chosen
- Toolchain configuration
- C/C++ version
- IntelliSense mode

**Comment by badsector998** [comment]
`c_cpp_properties.json` copasted from blink examples. Still not working.

**Comment by badsector998** [comment]
point C/C++ compilers to `arm-none-eabi-gcc.exe` then try build the project. Also include `${PICO_SDK_PATH}/**` in include files C/C++ extension config.

see `.vscode`

#2

**Comment by badsector998** [comment]
Closed #2

**Comment by badsector998** [comment]
#2
